stdlib/REPL/test/replcompletions.jl

```diff
@@ -87,7 +87,7 @@ end
 
 function map_completion_text(completions)
     c, r, res = completions
-    return map(completion_text, c), r, res
+    return unique!(map(completion_text, c)), r, res
```

Comment:
I think one of the intents of this test is to make sure that the completions aren't returning non-unique duplicates

Reply:
The completions can have duplicates which is why we call `unique!` on it in the REPL code itself, https://github.com/JuliaLang/julia/issues/28692, https://github.com/JuliaLang/julia/pull/28694.

Reply:
Yes, and this is the test you added in #28694 to check that the list does not have duplicates

Reply:
The test added there assumes that `test_completion` does the same as `complete_line`. However, right now, only `complete_line` does the `unique!` call. So the test is at least faulty.

Reply:
But I can see that with this change, the `count( ...) == 1` part is obsolete.

Reply:
Yes, that is right. The way things are written now, it is a bit hard to test this properly.

Reply:
> I think one of the intents of this test is to make sure that the completions aren't returning non-unique duplicates

Why would that be a problem ?